crates/rerun/src/run.rs

```diff
@@ -728,16 +730,14 @@ async fn run_impl(
                 let open_browser = args.web_viewer;
 
                 // This is the server that serves the Wasm+HTML:
-                let web_server_handle = tokio::spawn(host_web_viewer(
-                    args.bind.clone(),
+                host_web_viewer(
+                    &args.bind,
                     args.web_viewer_port,
                     args.renderer,
                     open_browser,
-                    _ws_server_url,
-                ));
-
-                // Wait for both servers to shutdown.
-                web_server_handle.await?.map_err(anyhow::Error::from)?;
+                    &_ws_server_url,
+                )?
+                .block_until_shutdown();
```

Comment:
Does the `ws_server` block in its destructor. or how does that handle shutdown?

Reply:
it's destructor will shut the server down. clarifying this in the code a bit